python/paddle/fluid/layers/nn.py

```diff
@@ -352,6 +353,38 @@ def fc(input,
     return helper.append_activation(pre_activation)
 
 
+def center_loss(input,
+                labels,
+                num_classes,
+                alpha,
```

Comment:
it is better to support both scalar and variable, and if a scalar is specified, we create a variable within this function

Reply:
done.